src/spec/spec.cr

```diff
@@ -209,7 +209,7 @@ OptionParser.parse! do |opts|
     exit
   end
   opts.on("-v", "--verbose", "verbose output") do
-    Spec.formatter = Spec::VerboseFormatter.new
+    Spec.formatters << Spec::VerboseFormatter.new
```

Comment:
@asterite Yeap. I guess is the best option.

Basically with `-v`:
`Spec.formatters = [ Spec::VerboseFormatter.new ]`

----

`I don't see how a user running specs would choose to add other formatters. I guess in their code that would have to do it, in a spec_helper.cr, right?`

Yes, the main idea is to be possible to use formatters to create `reports` (html/json), without affecting the terminal output, or even create something like: https://github.com/marceloboeira/spec2-nc.cr.